client/lib/products-values/test/index.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External Dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal Dependencies
+ */
+import {
+	JETPACK_PLANS,
+	PLAN_BUSINESS,
+	PLAN_FREE,
+	PLAN_PERSONAL,
+	PLAN_PREMIUM,
+} from 'lib/plans/constants';
+import {
+	isJetpackPlan,
+} from '..';
+
+/**
+ * Test helper to build a product object
+ *
+ * @param  {String} product_slug Product slug
+ * @return {Object}              Object containing product_slug
+ */
+const makeProductFromSlug = ( product_slug ) => ( { product_slug } );
+
+describe( 'products-values', () => {
```

Comment:
This `describe` should be omitted, because it'll be automatically added anyway:

![](https://cldup.com/V5Oyr6pX0d.png)